src/Mvc/Mvc.ViewFeatures/src/FormContext.cs

```diff
@@ -143,4 +158,20 @@ public void RenderedField(string fieldName, bool value)
 
         RenderedFields[fieldName] = value;
     }
+
+    internal bool InvariantField(string fieldName)
```

Comment:
nit: This one is clearer but maybe you could change to something like `IsInvariantField`?

Reply:
(as with the last two cases, this was for consistency)